examples/AN00239/README.rst

```diff
@@ -15,7 +11,8 @@ control of which debug messages are enabled.
 Required tools and libraries
 ............................
 
-.. appdeps::
+The CHANGELOG contains information about the current and previous versions.
```

Comment:
Since the section is titled 'Required tools and libraries', I do not see any need to reference the change log here.